### DISCUSSION TITLE
x86 Windows build malware discovery

### DISCUSSION THREAD

**Discussion by iiTool** [jopohl/urh#838] [Q&A] [answered]
I downloaded the most recent x86 Windows installer (2.9.1) and was notified during install that malware was detected in the urh.exe executable by Windows defender. I also ran it through virus total and 10/70 engines also detected the same Trojan in the urh.exe executable. I also downloaded the previous version (2.9.0) and found the same detections in urh.exe. I'm thinking it's a false positive but would be interested in what others have to say about this.

**Comment by jopohl** [comment, marked as answer]
Please see my answer [here](https://github.com/jopohl/urh/issues/804#issuecomment-702195611).
